etna/analysis/forecast/plots.py

```diff
@@ -196,7 +203,7 @@ def plot_forecast(
                             segment_borders_df.index.values,
                             values_low,
                             values_next,
-                            facecolor=forecast_color,
+                            color=forecast_color,
```

Comment:
So, this will set facecolor + edges. Right now, it is the same as before since we are filling only the area between. We might want to color edges a bit differently to give intervals a higher contrast.